cuppa/src/test/java/org/forgerock/cuppa/ModelTests.java

```diff
@@ -50,8 +50,10 @@ public void testBlockShouldBeImmutable() {
                 HookFunction.identity())))
                 .isExactlyInstanceOf(UnsupportedOperationException.class);
 
-        assertThatThrownBy(() -> testBlock.options.set(new TestOption("a")))
```

Comment:
This part isn't necessary any more. It was just asserting that the `Options` were immutable, which is always true now.